seeds/service.js

```diff
@@ -0,0 +1,103 @@
+const { Service } = require('../models');
+
+const serviceData = [
+  {
+    id: 1,
+    title: 'Salsa Dance class',
+    start: '2022-12-05T12:00:000Z',
```

Comment:
These dates did not insert into the database successfully, producing the following error: 
![image](https://user-images.githubusercontent.com/9697002/203881247-4d674956-c502-4f07-87b7-a7ce5037e372.png)


As discussed, can you please experiment with datetime formats as specified for mysql and described on w3schools here: https://www.w3schools.com/sql/sql_dates.asp

Please test these next time before submitting for the next review as discussed.